.github/scripts/build-images.sh

```diff
@@ -44,18 +44,37 @@ if (grep -q "${DF_PATH#./}" <<<$modified_files) || # Rebuild the image if any fi
     else
         echo "No extra arch is found, skipping install QEMU."
     fi
+    
```

Comment:
Why does your commit reorder these parts of the script rather than just making changes to the lines that it needs to touch?

Reply:
I would like to push fixed java images to the dockerhub and I noticed that the current script has a bug. We should build, push, and then manifest the images. But the implementation had placed manifest before push which is wrong. I changed the script to address this bug and also remove the replicated code between build and push parts. It is much shorter and simpler now.

Reply:
Please create another PR for that bug which you mentioned, a PR that just addresses that one bug only and we will have that merged before merging this PR.

Reply:
PR #311 includes the script change for the manifest issue and it's now merged into `dev`.
Please rebase your branch with the latest updates.

Reply:
This PR is rebased.